src/desktop/components/artist_page_cta/view.coffee

```diff
@@ -55,13 +55,15 @@ module.exports = class ArtistPageCTAView extends Backbone.View
   currentParams: ->
     qs.parse(location.search.replace(/^\?/, ''))
 
-  fullScreenOverlay: (e) =>
-    contextModule = "Footer" if e.currentTarget.className?.includes("artist-page-cta")
+  fullScreenOverlay: (event = "scroll") =>
```

Comment:
when would `event` not be set to the default "scroll" value as it seems like no arguments are being passed on line 43

Reply:
When the user clicks the footer banner which is also executes `fullScreenOverlay`. Since that wasn't immediately clear I refactored this a bit to pass the event object in `maybeShowOverlay` so we are alway operating off the event object.